packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.spec.tsx

```diff
@@ -70,9 +70,9 @@ describe('CustomViewLoader', () => {
       { projectKey }
     );
 
-    const iFrame = screen.getByTitle(
+    const iFrame = screen.getAllByTitle(
```

Comment:
Because now the drawer has its own title (which the previous custom panel didn't) and the iframe has a title, we need to select the correct one.

Reply:
No sure I understand this 🤔 
We did previously have a title on the Custom Panel, right?

![image](https://github.com/commercetools/merchant-center-application-kit/assets/97907068/569476d8-b487-4565-bb7e-493f80dfcb57)

Reply:
if you look at the code of the drawer, now the drawer has something like `<PageHeader>` which displays the title. the screenshot you have doesn't have that which is why we have 2 titles now.

Reply:
Thanks. Now I understand.